src/charm.py

```diff
@@ -58,15 +59,12 @@
     ActiveStatus,
     BlockedStatus,
     MaintenanceStatus,
-    ModelError,
     Relation,
     Secret,
     SecretNotFoundError,
     WaitingStatus,
 )
 from ops.pebble import ChangeError, Error, ExecError, Layer
-
-from hydra_cli import HydraCLI
 from utils import normalise_url, remove_none_values
```

Comment:
Import sorting seems to have changed, I think we should keep the current approach where local imports are separately grouped